.github/workflows/release-chart.yaml

```diff
@@ -31,3 +31,22 @@ jobs:
         env:
           CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
           CR_RELEASE_NAME_TEMPLATE: "metrics-server-helm-chart-{{ .Version }}"
+
+      - name: Get chart app version
+        id: chart_app_version
+        uses: mikefarah/yq@v4.12.2
+        with:
+          cmd: yq eval '.appVersion' './charts/metrics-server/Chart.yaml'
+
+      - uses: robinraju/release-downloader@v1.2
+        with:
+          repository: "kubernetes-sigs/metrics-server"
+          tag: "v${{ steps.chart_app_version.outputs.result }}"
+          fileName: "components.yaml"
+
+      - name: Update release
+        uses: ncipollo/release-action@v1
+        with:
+          allowUpdates: true
+          artifacts: "components.yaml"
```

Comment:
we will also need to upload the `high-availability.yaml` manifest as in: https://github.com/kubernetes-sigs/metrics-server/blob/master/.github/workflows/release.yaml#L16-L21

Reply:
thinking about it again after writing https://github.com/kubernetes-sigs/metrics-server/pull/826#discussion_r707543983, we won't be able to copy the `high-availability.yaml` manifest from the latest release until v0.6.0 is out

Reply:
It shouldn't be too hard to add this too for v0.6.0.